### PR TITLE
feat: remove custom hook for zerolog

### PR DESCRIPTION
**Pull request by lovung**
#### What type of PR is this?

- Remove custom hook function.
- Un-expose Otel keys.

#### What this PR does / why we need it (English/Chinese):

Unnecessary.

#### Which issue(s) this PR fixes:

N/A

### CODE REVIEW

**Comment by rogerogers** [comment]
The logger library already has a defined entry for the hook, I think so there is no need to add another one here.

**Comment by lovung** [comment]
> The logger library already has a defined entry for the hook, I think there is no need to add another one here.

@rogerogers OK. Then let me remove config.hookFn

**Comment by rogerogers** [comment]
In addition, the implementation of this hook form should reference the implementation of the logrus hook form. Here, overloading of the + hook is executed twice, thus this overloading is unnecessary and should be removed.

https://github.com/hertz-contrib/obs-opentelemetry/blob/8af103178b542475178f69100fed38f3b79b452f/logging/zerolog/logger.go#L58

**Comment by lovung** [comment]
@rogerogers I have just removed duplicate overloading.

File: logging/zerolog/logger.go [hunk 1]
```diff
@@ -32,9 +25,9 @@ type Logger struct {
...
 const (
-	TraceIDKey    = "trace_id"
-	SpanIDKey     = "span_id"
-	TraceFlagsKey = "trace_flags"
+	traceIDKey    = "trace_id"
+	spanIDKey     = "span_id"
+	traceFlagsKey = "trace_flags"
 )
 
 type ExtraKey string
```
Comment (rogerogers) [inline logging/zerolog/logger.go:33]: This ExtraKey has no use anymore.

Comment (lovung) [reply logging/zerolog/logger.go:33]: They are still being used in the default hook function.

Comment (rogerogers) [reply logging/zerolog/logger.go:33]: I searched, and it seems that it's no longer available. It has already been deleted.

Comment (rogerogers) [reply logging/zerolog/logger.go:33]: https://github.com/hertz-contrib/obs-opentelemetry/blame/8af103178b542475178f69100fed38f3b79b452f/logging/zerolog/option.go#L79-L107

Comment (lovung) [reply logging/zerolog/logger.go:33]: @rogerogers I just changed it's name

image

Comment (rogerogers) [reply logging/zerolog/logger.go:33]: I mean `type ExtraKey string`.

Comment (lovung) [reply logging/zerolog/logger.go]: I see. Just removed.

**Review by rogerogers — APPROVED** [review]

**Pull request merged**